API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
Which sentence talks about compile above it? 

![screen shot 2015-12-25 at 21 42 46](https://cloud.githubusercontent.com/assets/26678/12005266/7fc62264-ab50-11e5-817c-b0521a8945ca.png)